rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -4,85 +4,88 @@ Create and document the environment for the automation of processes.
+Project managers organize projects into categories. A category describes the environment of a process in a few keywords, for example `Customer Support Mystical University` or `Finance`.
 
-A project can only be assigned to a single category. First create a standard category, so you can start with process automation quickly. You can change the category of a process later. After you have automated the first projects, it makes sense to create an archive category and move completed projects there.
+A project can only belong to a single category. First, create a standard category, so the process team can start with process automation quickly. They can change the category of a process later. After they have automated the first few projects, create an archive category to move the completed projects there.
 
 ==== Create a Category
 
-Create categories to group processes. Assign a process to a category during evaluation or project creation.
+Create categories for the organization of processes:
 
-. In the *Process Management* view of the *Process Automation* module, click *Create*.
+. Open the *Process Management* view of the *Process Automation* module.
+. Click *Create*.
 . Enter a name in the *Create a category* window.
 . Click *OK*.
 
 ==== Edit a Category
 
-Edit a category to render its name more precisely.
+Edit a category to render its name more precisely:
 
-. In the *Process Management* view of the *Process Automation* module, click the *Edit* icon image:edit-icon.png[pen-to-square symbol,1.5%,1.5%] on the panel of the category you want to edit.
+. Open the *Process Management* view of the *Process Automation* module.
+. Click the *Edit* icon image:edit-icon.png[pen-to-square symbol,1.5%,1.5%] on the panel of the category you want to edit.
 . Change the name in the *Edit the category* window.
 . Click *OK*.
 
 ==== Remove a Category
 
-Remove categories you no longer need.
-
-Only categories that do not contain processes can be deleted.
+Remove categories that are no longer needed. You can only remove categories that do not contain processes:
 
-. In the *Process Management* view of the *Process Automation* module, click the the *Delete* icon image:delete-icon.png[trash symbol,1.5%,1.5%] on the panel of the category you want to delete.
+. Open the *Process Management* view of the *Process Automation* module.
+. Click the the *Remove* icon image:delete-icon.png[trash symbol,1.5%,1.5%] on the panel of the category you want to delete.
 
 === Manage Applications
 
-In the view *Process Applications*, list all applications required for the execution of processes. In the view *Process Management*, assign required applications to processes at process creation or phase transition. In the module *Bot Management*, assign applications to a bot that can execute them. Bots can execute all applications installed on their computer.
+In the *Process Applications* view, you list all applications required for the execution of processes.
```

Comment:
Maybe you can do without the "you" and start the sentence by saying "List all applications required for...".

Reply:
same as above